### PR TITLE
Auth login

### PR DESCRIPTION
**Pull request by EstebanIndiveri**
in this commit I added a custom tabBar to navigate, added illustration and icons, I used one stack to visitor with two tabs and a complete stack with 5 tabs to authenticated user 

#### Android
Captura de Pantalla 2021-07-13 a la(s) 20 12 29

Captura de Pantalla 2021-07-14 a la(s) 16 05 25

Captura de Pantalla 2021-07-13 a la(s) 20 14 17

untitlednav

#### IOS
Captura de Pantalla 2021-07-13 a la(s) 21 52 01
Captura de Pantalla 2021-07-13 a la(s) 21 52 01
Captura de Pantalla 2021-07-13 a la(s) 22 23 41
Simulator Screen Recording - iPhone 11 - 2021-07-13 at 22 24 13

### CODE REVIEW

File: src/app/components/TabBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import Routes from '@constants/routes';
+import React, { useEffect, useState } from 'react';
+import { View, ImageSourcePropType, Image } from 'react-native';
+import IC_HOME_INACTIVE from '@assets/TabBar/ic_home_inactive.png';
```
Comment (santiagorobra) [inline src/app/components/TabBar/index.tsx:4]: in projects we usually use camel and lower to name files and variables

File: src/app/components/TabBar/styles.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { gray, green, white } from '@constants/colors';
+import { StyleSheet } from 'react-native';
```
Comment (santiagorobra) [inline src/app/components/TabBar/styles.ts:2]: first react native

Comment (santiagorobra) [reply src/app/components/TabBar/styles.ts:2]: move up

File: src/app/components/TabBar/styles.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+const styles = StyleSheet.create({
+  image: {
+    height: 28,
+    width: 30,
+    marginTop: 3
+  },
+  textActive: {
```
Comment (santiagorobra) [inline src/app/components/TabBar/styles.ts:10]: You are using it?

File: src/app/components/TabBar/styles.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    color: green
+  },
+  textInactive: {
```
Comment (santiagorobra) [inline src/app/components/TabBar/styles.ts:13]: same

File: src/app/components/TabBar/styles.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    color: gray
+  },
+  selectedRoute: {
+    padding: 14,
+    backgroundColor: white,
+    marginBottom: 10,
+    borderRadius: 100
```
Comment (santiagorobra) [inline src/app/components/TabBar/styles.ts:20]: 100 is a lot, sure or you need that much?

File: src/app/components/TabBar/index.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
+import Routes from '@constants/routes';
```
Comment (santiagorobra) [inline src/app/components/TabBar/index.tsx:1]: move down

File: src/app/screens/Auth/screens/Login/components/Header/index.tsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Image } from 'react-native';
+import HEADER from '@assets/illustrations/headerLogin.png';
```
Comment (santiagorobra) [inline src/app/screens/Auth/screens/Login/components/Header/index.tsx:3]: If it is an illustration, the name has to start with 'il' 
for example ilHeaderLogin.png

File: src/app/screens/Auth/screens/Login/styles.ts [hunk 8]
```diff
@@ -11,10 +11,49 @@ export default StyleSheet.create({
...
+    color: green,
+    width: 290,
+    textAlign: 'left'
+  },
   formButton: {
     backgroundColor: green,
     padding: 10,
-    borderRadius: 3,
-    marginTop: 15
+    borderRadius: 25,
```
Comment (santiagorobra) [inline src/app/screens/Auth/screens/Login/styles.ts:47]: if values ​​are repeated you could create a variable

for example
COMMON_VALUE_25 = 25

File: src/app/screens/Auth/screens/SignUp/styles.ts [hunk 9]
```diff
@@ -1,23 +1,61 @@
...
+    width: 290,
+    textAlign: 'left'
   },
   formButton: {
     backgroundColor: green,
-    borderRadius: 3,
-    marginTop: 15,
-    padding: 10
+    padding: 10,
+    borderRadius: 25,
```
Comment (santiagorobra) [inline src/app/screens/Auth/screens/SignUp/styles.ts:49]: same

File: src/config/navigation.tsx [hunk 10]
```diff
@@ -1,18 +1,36 @@
...
 import fonts from './fonts';
 
+const heightTabNav = isIos ? 70 : 55;
+const paddingButtonTabNav = isAndroid ? 20 : 25;
+const styles = StyleSheet.create({
+  styleTabNav: {
+    height: heightTabNav,
+    paddingTop: 5,
+    position: 'absolute',
+    borderTopLeftRadius: 25,
```
Comment (santiagorobra) [inline src/config/navigation.tsx:18]: same

File: src/config/navigation.tsx [hunk 11]
```diff
@@ -57,16 +76,16 @@ export const appScreensNavOptions = {
...
 };
 
 export const statusBarStyles = {
   // TODO: Change these styles to customize the status bar
-  [Routes.Login]: statusBarConfig.whiteStatusBar,
-  [Routes.SignUp]: statusBarConfig.blueStatusBar,
-  [Routes.Tab1]: statusBarConfig.blueStatusBar,
-  [Routes.Tab2]: statusBarConfig.blueStatusBar,
-  [Routes.Home]: statusBarConfig.blueStatusBar,
+  [Routes.Login]: statusBarConfig.greenStatusBar,
```
Comment (santiagorobra) [inline src/config/navigation.tsx:85]: is not it better to be the default? greenStatusBar

File: src/constants/colors.ts [hunk 12]
```diff
@@ -1,8 +1,9 @@
 export const blue: string = '#288CCB';
-export const green: string = '#53E69D';
+export const green: string = '#5CB85C';
```
Comment (santiagorobra) [inline src/constants/colors.ts:2]: add a TODO, andy's PR already added this color

File: src/constants/colors.ts [hunk 13]
```diff
@@ -1,8 +1,9 @@
...
 export const red: string = '#FF0000';
 export const transparent: string = 'transparent';
 export const white: string = '#FFF';
 export const black: string = '#000';
 export const gray: string = '#rgba(0, 0, 0, 0.38)';
 export const darkGray: string = '#363636';
+export const lightGray: string = '#e1e1e1';
```
Comment (santiagorobra) [inline src/constants/colors.ts:9]: uppercase

File: src/app/components/TabBar/index.tsx (hunk 1 above)
Comment (EstebanIndiveri) [reply src/app/components/TabBar/index.tsx]: changed

File: src/app/components/TabBar/styles.ts (hunk 2 above)
Comment (EstebanIndiveri) [reply src/app/components/TabBar/styles.ts:1]: moved

File: src/app/components/TabBar/styles.ts (hunk 4 above)
Comment (EstebanIndiveri) [reply src/app/components/TabBar/styles.ts]: right, removed

File: src/app/components/TabBar/styles.ts (hunk 5 above)
Comment (EstebanIndiveri) [reply src/app/components/TabBar/styles.ts:14]: I think it like circle in center tab, if you believe this is unnecessary, i can removed

Comment (santiagorobra) [reply src/app/components/TabBar/styles.ts:14]: no this is fine, I resolve the comment

File: src/app/components/TabBar/styles.ts (hunk 3 above)
Comment (EstebanIndiveri) [reply src/app/components/TabBar/styles.ts]: removed

File: src/app/components/TabBar/index.tsx (hunk 6 above)
Comment (EstebanIndiveri) [reply src/app/components/TabBar/index.tsx]: moved

File: src/app/screens/Auth/screens/Login/components/Header/index.tsx (hunk 7 above)
Comment (EstebanIndiveri) [reply src/app/screens/Auth/screens/Login/components/Header/index.tsx]: changed with correct name

File: src/app/screens/Auth/screens/Login/styles.ts (hunk 8 above)
Comment (EstebanIndiveri) [reply src/app/screens/Auth/screens/Login/styles.ts]: added const

File: src/config/navigation.tsx (hunk 10 above)
Comment (EstebanIndiveri) [reply src/config/navigation.tsx]: added const value

File: src/app/screens/Auth/screens/SignUp/styles.ts (hunk 9 above)
Comment (EstebanIndiveri) [reply src/app/screens/Auth/screens/SignUp/styles.ts]: const with value 25

File: src/config/navigation.tsx (hunk 11 above)
Comment (EstebanIndiveri) [reply src/config/navigation.tsx]: replace with default

File: src/constants/colors.ts (hunk 12 above)
Comment (EstebanIndiveri) [reply src/constants/colors.ts:3]: added TODO with green color idk what its correct

File: src/constants/colors.ts (hunk 13 above)
Comment (EstebanIndiveri) [reply src/constants/colors.ts]: changed

**Review by santiagorobra — APPROVED** [review]

**Pull request merged**